json-schemas/workflowRunsRequest.json

```diff
@@ -146,6 +146,9 @@
                     }
                 }
             }
+        },
+        "authenticityToken": {
```

Comment:
What's the NextGen equivalent, or is there no NextGen equivalent (getting replaced by the auth stuff that @jfoo1984's doing?)?

If it's Rails only I think we should stick it into `todoRemove`.